bundler/lib/bundler/cli/init.rb

```diff
@@ -32,7 +32,7 @@ def run
           file << spec.to_gemfile
         end
       else
-        File.open(File.expand_path("../templates/#{gemfile}", __dir__), "r") do |template|
+        File.open(File.expand_path(template_path, __dir__), "r") do |template|
```

Comment:
We currently have two gemfile templates: `gems.rb` and `Gemfile`, but they have the same content. I think we can use `../templates/Gemfile` unconditionally here and remove the `gems.rb` template and the `template_path` method?

Reply:
We can. I'll do it.